neo/SmartContract/ContractParameter.py

```diff
@@ -0,0 +1,130 @@
+from neo.SmartContract.ContractParameterType import ContractParameterType, ToName
+from neo.VM.InteropService import StackItem, Array, ByteArray, Struct, Boolean, Integer, InteropInterface
+import binascii
+from neocore.UInt160 import UInt160
+from neocore.UInt256 import UInt256
+from neocore.BigInteger import BigInteger
+from neocore.Cryptography.ECCurve import ECDSA
+
+
+class ContractParameter():
+
+    Type = None
+    Value = None
+
+    def __init__(self, type, value):
+        self.Type = type
+        self.Value = value
+
+    @staticmethod
+    def Parse0x(param):
+        if param[0:2] == '0x':
+            return param[2:]
+        return param
+
+    @staticmethod
+    def ParamToUInt160(param):
+        shash_reversed = bytearray(binascii.unhexlify(ContractParameter.Parse0x(param)))
+        shash_reversed.reverse()
+        return UInt160(data=shash_reversed)
+
+    @staticmethod
+    def ParamToUInt256(param):
+        shash_reversed = bytearray(binascii.unhexlify(ContractParameter.Parse0x(param)))
```

Comment:
Should this perhaps be a utility function in a more generic class or maybe even `neocore`? `UInt256` would be used for `Transaction`s beyond smart contracts (e.g. reversing the assetid for an transaction output entry, or handling the transaction id for an input entry)